jetstream/options.go

```diff
@@ -301,23 +317,26 @@ func WithExpectStream(stream string) PublishOpt {
 	}
 }
 
-// WithExpectLastSequence sets the expected sequence in the response from the publish.
+// WithExpectLastSequence sets the expected sequence in the response from the
+// publish.
 func WithExpectLastSequence(seq uint64) PublishOpt {
 	return func(opts *pubOpts) error {
 		opts.lastSeq = &seq
 		return nil
 	}
 }
 
-// WithExpectLastSequencePerSubject sets the expected sequence per subject in the response from the publish.
+// WithExpectLastSequencePerSubject sets the expected sequence per subject in
+// the response from the publish.
 func WithExpectLastSequencePerSubject(seq uint64) PublishOpt {
 	return func(opts *pubOpts) error {
 		opts.lastSubjectSeq = &seq
 		return nil
 	}
 }
 
-// WithExpectLastMsgID sets the expected last msgId in the response from the publish.
+// WithExpectLastMsgID sets the expected last msgId in the response from the
+// publish.
```

Comment:
can also be clarified - I think this is about the Nats-Message-Id header in the most recent message in the stream?

Reply:
Changed to:
```go
// WithExpectLastMsgID sets the expected message ID the last message on a stream
// should have. If the last message has a different message ID server will
// reject the message and publish will fail.
```